catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -498,6 +501,14 @@ public void setIngest(Boolean isIngest) {
         this.isIngest = isIngest;
     }
 
+    public Boolean getAllowEditing() {
```

Comment:
Not sure if its necessary or just good practice but for booleans I think the getter goes like `isBla` so this would be `isAllowEditing`